metacatalog/ext/export_standard/schemas/iso19115/iso19115-2.j2

```diff
@@ -0,0 +1,412 @@
+        {# no datasource: utf8 #}
+        <gmd:MD_CharacterSetCode codeListValue="utf8" codeList="http://www.isotc211.org/2005/resources/codeList.xml#MD_CharacterSetCode"/>
+    {% endif %}
+    </gmd:characterSet>
+    {% if isPartial %}
+    {# if Entry isPartial: ids of associated groups #}
+    <gmd:parentIdentifier>
+        {# parentIdentifier: EntryGroup.id / EntryGroup.uuid ?? #}
+        {% for group in associated_groups %}
+        <gco:CharacterString>{{group.uuid}}</gco:CharacterString>
+        {% endfor %}
+    </gmd:parentIdentifier>
+    {% endif %}
+    <gmd:hierarchyLevel>
+        {# always dataset #}
+        <gmd:MD_ScopeCode codeList="http://www.isotc211.org/2005/resources/Codelist/gmxCodelists.xml#MD_ScopeCode" codeListValue="dataset" codeSpace="005">dataset</gmd:MD_ScopeCode>
+    </gmd:hierarchyLevel>
+
+    {# this would be our contact (pointOfContact) #}
+    <gmd:contact xlink:title="DOC/NOAA/NESDIS/NODC/NCDDC&gt; National Coastal Data Development Center (pointOfContact)">
+        <gmd:CI_ResponsibleParty>
+            <gmd:organisationName>
+                <gco:CharacterString>Karlsruhe Institute for Technology (KIT) - Institute of Water and River Basin Management - Hydrology</gco:CharacterString>
+            </gmd:organisationName>
+            <gmd:contactInfo>
+                <gmd:CI_Contact>
+                    <gmd:address>
+                        <gmd:CI_Address>
+                            <gmd:deliveryPoint>
+                                <gco:CharacterString>Building 10.81</gco:CharacterString>
+                            </gmd:deliveryPoint>
+                            <gmd:city>
+                                <gco:CharacterString>Karlsruhe</gco:CharacterString>
+                            </gmd:city>
+                            <gmd:administrativeArea>
+                                <gco:CharacterString>Baden Wurttemberg</gco:CharacterString>
+                            </gmd:administrativeArea>
+                            <gmd:postalCode>
+                                <gco:CharacterString>76131</gco:CharacterString>
+                            </gmd:postalCode>
+                            <gmd:country>
+                                <gco:CharacterString>Germany</gco:CharacterString>
+                            </gmd:country>
+                            <gmd:electronicMailAddress>
+                                <gco:CharacterString>alexander.dolich@kit.edu</gco:CharacterString>
+                            </gmd:electronicMailAddress>
+                            <gmd:electronicMailAddress>
+                                <gco:CharacterString>mirko.maelicke@kit.edu</gco:CharacterString>
+                            </gmd:electronicMailAddress>
+                        </gmd:CI_Address>
+                    </gmd:address>
+                    <gmd:hoursOfService>
+                        <gco:CharacterString>Monday - Friday 9:00am - 5:00pm</gco:CharacterString>
```

Comment:
I am not sure, who to put here. 
My suggestion: For the metacatalog package we should get rid of all KIT specific settings and put dummy adresses there. 
Then, there needs to be something like a `template='/home/mytemplates'` attribute, to overwrite the place, where the jinja file can be found. alternatively, we use the same folder and put in a `vforwater.j2`, which we also add to the gitignore.